setup.py

```diff
@@ -1,92 +1,28 @@
 #!/usr/bin/env python
 # -*- encoding: utf-8 -*-
-from __future__ import absolute_import
-
-import io
-import re
-from glob import glob
-from os import path, environ
-
-from setuptools import find_packages
+import os
 
 # Include extensions only when not on readthedocs.org
-if environ.get('READTHEDOCS', None) == 'True':
+if os.environ.get('READTHEDOCS', None) == 'True':
     from setuptools import setup
     from distutils.core import Extension
     extensions = []
 else:
     from numpy.distutils.core import setup, Extension
+
```

Comment:
Apexpy worked fine on windows before, why is this needed suddenly?

Reply:
The install silently fails on windows.